opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1998,8 +1998,12 @@ namespace Opm
             return;
         }
 
-        if (!group.isInjectionGroup())
+        if (!group.isInjectionGroup() || currentGroupControl == Group::InjectionCMode::NONE) {
+            // use bhp as control eq and let the updateControl code find a vallied control
```

Comment:
Typo: valid
(also line 2128, and in StandardWell_impl.hpp)

Reply:
Done